src/components/Users/Users.js

```diff
@@ -47,8 +49,44 @@ const Users = (props) => {
       </div>
       <div>
         {user.followed
-          ? <button onClick={() => props.unfollow(user.id)}>Unfollow</button>
-          : <button onClick={() => props.follow(user.id)}>Follow</button>}
+
+          ? (
+            <button onClick={() => {
+              axios.delete(`${USERS_URL}follow/${user.id}`, {
+                withCredentials: true,
+                headers: {
+                  'API-KEY': 'a330ffbb-7ace-4dc9-a8ec-7880ecab78ff',
```

Comment:
Consider storing the API key in the core level constants.

Reply:
/fxd